larq_compute_engine/mlir/python/saved_model_tfl_flatbuffer.cc

```diff
@@ -0,0 +1,162 @@
+#include <exception>
```

Comment:
Since this file is directly adapted from a TF file, I think we need to include [the license comment](https://github.com/tensorflow/tensorflow/blob/4ae0971abe3ad5c3a43c5c79283c98cfb08f4bdc/tensorflow/compiler/mlir/lite/python/saved_model_to_tfl_flatbuffer.cc#L1-L14).

Reply:
I wasn't aware that this was directly adapted from TF. I added a license comment.